crates/re_viewer/build.rs

```diff
@@ -43,16 +44,26 @@ macro_rules! bail {
 }
 
 fn git_branch_name(sh: &Shell) -> Result<String> {
-    Ok(String::from_utf8(
-        cmd!(sh, "git rev-parse --abbrev-ref HEAD").output()?.stdout,
-    )?)
+    let mut branch_name =
+        String::from_utf8(cmd!(sh, "git branch --show-current").output()?.stdout)?;
+    branch_name.truncate(branch_name.trim_end().len()); // trim trailing whitespace in-place
+    Ok(branch_name)
 }
```

Comment:
We already have this function in `crates/re_build_tools/src/git.rs` - we should de-duplicate this code

Reply:
Same with `git_short_hash` - it should be moved to `crates/re_build_tools/src/git.rs`